rootfs/api/models.py

```diff
@@ -790,9 +747,25 @@ class Meta:
         ordering = ['-created']
         unique_together = (('app', 'uuid'),)
 
+    @property
+    def _scheduler(self):
```

Comment:
Is there somewhere that calls this? If we're being proper here only an application type should have a scheduler client attached to it